README.md

```diff
@@ -192,6 +192,44 @@ machine. Just install mingw-w64 and then
 $ make
 ```
 
+## bash/zsh completion
+
+If [`genzshcomp`][python-genzshcomp] is installed, `make` generates shell completion files for bash (bash-completion) and zsh.
+
+They enable completion of available options.
```

Comment:
```suggestion
They enable tab completions of available command line options.
```

Reply:
Googled but "tab completion*s* of available" was not found.